src/dev/flang/fuir/analysis/dfa/DFA.java

```diff
@@ -1910,6 +1910,14 @@ static void setArrayElementsToAnything(Call cl, int argnum, String intrinsicName
         jobj.setField(cl._dfa, jref, Value.UNKNOWN_JAVA_REF);
         return jobj;
       });
+    put("fuzion.java.null_to_java_object", cl ->
+      {
+        var rc = cl._dfa._fuir.clazzResultClazz(cl._cc);
```

Comment:
@maxteufel what if you replace all of this up to return by newFuzionJavaCall(cl) as in `fuzion.java.call_c0`